examples/fractal_music/sketch.js

```diff
@@ -1,94 +1,109 @@
+/**
+ *  Example: Fractal Music
+ *  Generate a fractal pattern of music using L-Systems. 
+ *  We begin with an initial sequence of tokens, and use a programmer-
+ *  defined set of rules to generate new tokens. Here, we apply the rules:
+ *  + > + -
+ *  - > +
+ *  to each token in existing sequences to generate new sequences. The 
+ *  tokens used here are "+","-" which steer the melody's pitch upward or 
+ *  downward. Note durations are selected randomly.
+ */
+
 var rules = {
-  "A":["A","B"],
-  "B":["E","C"],
-  "C":["F","G"],
-  "D":["D","F"],
-  "E":["G"],
-  "F":["B","A"],
-  "G":["C"]
+  "+":["+","-"],
+  "-":["+"]
```

Comment:
could we use numbers to represent these intervals?

semitone intervals are not so pleasant, have you tried whole tone, fourth, fifth?

this pattern will wind up increasing, how to address this?